pkgs/by-name/as/ascii-weather/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  fetchFromGitHub,
+  lib,
+  ncurses,
+  gccStdenv,
+}:
+gccStdenv.mkDerivation {
```

Comment:
After looking at the codebase, there's really nothing GCC-specific that prevents it from working with Clang, aside from the Makefile forcing it to be used (`CC = gcc`).

Can you change the Makefile so that it respects the CC set by the build system? We really would like to avoid requiring GCC on Darwin if possible.